src/ConstructiveSolidGeometry/plotting/CSG/CSG.jl

```diff
@@ -6,10 +6,12 @@ function primitives(csg::AbstractConstructiveGeometry)
     ps
 end
 
-@recipe function f(csg::AbstractConstructiveGeometry{T}; n_samples = 40) where {T}
+@recipe function f(csg::AbstractConstructiveGeometry{T}; n_samples = 40, CSG_scale = missing) where {T}
     seriestype --> :csg
     ps = primitives(csg)
-    spacing = (haskey(plotattributes, :seriestype) && plotattributes[:seriestype] == :samplesurface) ?  T(get_scale(csg)/n_samples) : nothing
+    if haskey(plotattributes, :seriestype) && plotattributes[:seriestype] == :samplesurface
+        spacing::T = T((ismissing(CSG_scale) ? get_scale(csg) : CSG_scale)/n_samples)
```

Comment:
Now, the scales are only determined once for a detector and passed to the plot recipes of the individual geometries.
However, if the geometries/objects are plotted by themselves, the scales are determined within that